src/main/java/org/jenkinsci/maven/plugins/hpi/TestDependencyMojo.java

```diff
@@ -517,8 +517,7 @@ private static void appendEntries(String property, Collection<String> additions,
      * @param war The WAR to scan.
      * @return The bundled plugins in the WAR.
      */
-    private static Map<String, String> scanWar(File war, MavenSession session, MavenProject project)
-            throws MojoExecutionException {
+    private Map<String, String> scanWar(File war) throws MojoExecutionException {
```

Comment:
`getLog()` is set for the mojo object, so this needs to be non-static.  the `MavenSession` and `MavenProject` are already set in the Mojo - so these are used directly.